docs/topics/readme/zip-health-check-nodejs.properties

```diff
@@ -6,4 +6,6 @@ app-name=booster-health-check-nodejs
 gitCMD=unzip ${artifactId}.zip
 targetRepository=${artifactId}
 cicdSection=
-prerequisite=IMPORTANT: This booster requires Node.js 8.x or greater and `npm` 5 or greater.
\ No newline at end of file
+prerequisite=IMPORTANT: This booster requires Node.js 8.x or greater and `npm` 5 or greater.
+integrationTestSection=
+integrationTestSection=NOTE: Run the set of integration tests included with this booster using `mvn clean verify -Popenshift,openshift-it`.
```

Comment:
Only the 1st `integrationTestSection` line is correct.

Reply:
Thanks, removed the incorrect one.